public/admin/bower_components/bootstrap-social/README.md

```diff
@@ -0,0 +1,58 @@
+Social Buttons for Bootstrap
+============================
```

Comment:
Please do not commit the bower_components (they should be installed through the bower package manager).